src/components/SkipLink.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import styled from 'styled-components';
+import { LinkProps, Link, Typography } from '@material-ui/core';
+
+const StyledSkipLink = styled(Link)`
+  position: absolute;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 3rem;
+  background: ${({ theme }) => theme.palette.section.black};
+
+  :focus {
+    position: static;
+  }
+`;
+
+const StyledTypography = styled(Typography)`
+  color: ${({ theme }) => theme.palette.background.default};
+  outline: 3px solid ${({ theme }) => theme.palette.error.main};
```

Comment:
dette har jo ikke noe med error å gjøre. men fargen ligner på det som brukes på siden til uutilsynet, så jeg er villig til å se mellom fingrene :) føler det er overkill å utvide theme med ny property bare for omrisset her

Reply:
men må omrisset være rødt da?

Reply:
blir det ikke mer naturlig å bruke noen av våre farger? (selv om de ikke kan se eller ser dårlig) 😅  jeg ville tenkt at det hadde vært naturlig å bruke den gulfargen vi bruker her

Reply:
det kan vi  forsåvidt - godt poeng!
jeg har egentlig bare tatt fra DLR som er inspirert av uutilsynet